src/store.js

```diff
@@ -1,4 +1,15 @@
 import {writable} from 'svelte/store'
 import * as j from './test.json'
 export const language = writable('en');
-export const json = writable(j);
\ No newline at end of file
+export const json = writable(j);
+
+// test data from project json for resource viewer
+import * as data from './wordweb.json'
+// makes data from a specific available to the outside
+export const project_json = writable(data);
```

Comment:
I would suggest to remove this and just work with one json at the time.

Reply:
I agree to remove these lines but prefer to keep the project files (wordweb.json, pou.json etc.) so we have several test possiblities